es6-classes/square.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(width) {
+    super();
+    this.width = width;
+    this.area = this.width * this.width;
+    this.perimeter = this.width * 4;
```

Comment:
You should not be creating these properties manually. You should be utilizing the `super` function to create these